Resources/public/js/views/subitem/ez-subitemgridview.js

```diff
@@ -103,9 +103,9 @@ YUI.add('ez-subitemgridview', function (Y) {
                     this._disableLoadMore();
                 }
             });
-            this.after('subitemsChange', function () {
+            this.after('subitemsChange', function (e) {
                 this._uiUpdatePagination();
-                this._appendGridItem();
+                this._appendGridItem(e);
```

Comment:
instead of passing the EventFacade, here you should compute an array containing the newly added subitems and then pass this to `_appendGridItem` which should stay focused on appending grid item as its name suggests.